types/block_meta.go

```diff
@@ -2,17 +2,19 @@ package types
 
 // BlockMeta contains meta information.
 type BlockMeta struct {
-	BlockID BlockID `json:"block_id"`
-	Header  Header  `json:"header"`
-	NumTxs  int64   `json:"number_txs"`
+	BlockID   BlockID `json:"block_id"`
+	BlockSize int     `json:"block_size"`
```

Comment:
Do we care about the ordering here re breaking? Ie could we consider this "less breaking" if we just add fields to the end instead of in the middle?

Reply:
In the presence of other protocol breaking changes? Probably no